pkgs/applications/video/ani-cli/default.nix

```
@@ -0,0 +1,41 @@
+{ fetchFromGitHub, makeWrapper, stdenv, config, lib, gnugrep, gnused, curl
+, openssl, mpv, aria2 }:
```

```suggestion
{ fetchFromGitHub
, makeWrapper
, stdenv
, lib
, gnugrep
, gnused
, curl
, openssl
, mpv
, aria2
}:
```

Formatting aside, removed unused `config` input. See previous comment regarding `stdenv` or `stdenvNoCC`.